AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2259,15 +2310,7 @@ - (CGSize)calculatedSize
 - (ASSizeRange)constrainedSizeForCalculatedLayout
 {
   ASDN::MutexLocker l(__instanceLock__);
-  return _calculatedLayout.constrainedSizeRange;
-}
-
-- (void)setLayoutSpecBlock:(ASLayoutSpecBlock)layoutSpecBlock
```

Comment:
Add `layoutSpecThatFits:` getter / setter back in with lock. Don't know why this were removed though ...